test/runtests.jl

```diff
@@ -21,6 +21,53 @@ function draw2(μ)
     return (x,y)
 end
 
+measures = [
+    # Chain(x -> Normal(μ=x), Normal(μ=0.0))
+    For(3) do j Normal(σ=j) end
+    For(2,3) do i,j Normal(i,j) end
+    # Likelihood
+    # Pointwise
+    Normal() ^ 3
+    Normal() ^ (2,3)
+    # SpikeMixture(Normal(), 2)
+    # Dirac(0.0) + Normal()
+    # transforms
+    3 * Normal()
+    Bernoulli(0.2)
+    Beta(2,3)
+    Binomial(10,0.3)
+    Cauchy()
+    Dirichlet(ones(3))
+    Exponential()
+    Gumbel()
+    # InverseGamma(2)
+    Laplace()
+    LKJCholesky(3,2.0)
+    Multinomial(n=10,p=[0.2,0.3,0.5])
+    # MvNormal
+    NegativeBinomial(5,0.2)
+    Normal(2,3)
+    Poisson(3.1)
+    StudentT(ν=2.1)    
+    Uniform()
+    # CountingMeasure(Float64)
+    Dirac(π)
+    Lebesgue(ℝ)
```

Comment:
What about `Lebesgue` with some other supports?

Reply:
There's not yet anything else that works. For higher dimensions, we currently do `Lebesgue(ℝ) ^ n`, though I think I like @gdalle 's idea of `EuclideanSpace` better.

Also, the whole `AbstractDomain` thing was just to get things going, and to avoid committing to a particular representation like `DomainsSets` that would bring additional dependencies. So maybe `AbstractDomain` is fine and just needs some more spaces added, or we could rethink how this is set up. In any case I'm open to feedback on the design.

Reply:
Btw, I had doubts about the `testvalue` of a multidimensional `Lebesgue` measure. The idea behind using `StaticVector`s was that their size was encoded in the type: otherwise, all multi-dimensional `Lebesgue` measures will have the same `testvalue` type. Is that a problem?